### PR TITLE
Seed nodes

### PR DESCRIPTION
**Pull request by aguycalled**
This Pull Request updates the mainnet and testnet nodes in chainparamsseeds.h.

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -400,10 +400,10 @@ class CTestNetParams : public CChainParams {
         nPruneAfterHeight = 1000;
         bnProofOfWorkLimit = arith_uint256(~arith_uint256() >> 16);
     
-        uint32_t nTimestamp = 1545341312;
-        uint256 hashGenesisBlock = uint256S("0x0000a2ed763c6efc24bbb3ac8d9f1ab9e8f1e7100d5221ad80815cd7b369dc2c");
-        uint256 hashMerkleRoot = uint256S("0x02128838f2516796eb04f5b3fd143a7786001301dc5ffcfd2b2c687a2864aae9");
-        uint32_t nNonce = 2043585747;
+        uint32_t nTimestamp = 1573858751;
```
Comment (proletesseract) [inline src/chainparams.cpp]: IDK why this diff is here when PR #628 as been merged?

**Comment by navbuilder** [comment]
A new build of ${LASTCOMMIT} has completed succesfully!
Binaries available at https://build.nav.community/binaries/${arrPR[1]}

File: src/chainparams.cpp (hunk 1 above)
Comment (mxaddict) [reply src/chainparams.cpp]: @proletesseract somehow the diff did not update, I merged master into this PR and it seems to have fixed the diff on github.

**Comment by mxaddict** [comment]
@aguycalled how did you generate the list of IPs in txt and in the header file? I'm curios.

**Comment by aguycalled** [comment]
`cd contrib/seeds/ && python generate-seeds.py .`

**Comment by mxaddict** [comment]
> `cd contrib/seeds/ && python generate-seeds.py .`

Should we maybe make this a step in the default release process?

So that each new release has the latest seeds.

**Comment by aguycalled** [comment]
yes, at least for every minor release

**Comment by navbuilder** [comment]
A new build of 51f7ed798a8a1e89045f6f71666e4a73edb2a126 has completed succesfully!
Binaries available at https://build.nav.community/binaries/seed_nodes

**Comment by proletesseract** [comment]
Building

**Comment by proletesseract** [comment]
Compiles and runs on OSX. Immediately gets a connection.

**Pull request merged**